Okta.Core.Tests/Clients/UsersClientTests.cs

```diff
@@ -464,24 +484,40 @@ public void GetUserByUsername()
             catch (OktaException e)
             {
                 string strEx = string.Format("Error Code: {0} - Summary: {1} - Message: {2}", e.ErrorCode, e.ErrorSummary, e.InnerException.InnerException.Message);
-                //Console.WriteLine(strEx);
+                Console.WriteLine(strEx);
             }
-
         }
 
- 
-        //private Tenant getTenant(TestContext context)
-        //{
-        //    Tenant tenant = new Tenant
-        //    {
-        //        Id = Convert.ToInt32(context.DataRow["Id"]),
-        //        Url = Convert.ToString(context.DataRow["Url"]),
-        //        ApiKey = Convert.ToString(context.DataRow["ApiKey"])
+        [TestMethod]		
+         public void SetCredentials()
+        {
+            string strUserID = oktaTenant.TestUserId;
 
-        //    };
+            try
+            {
+                var usersClient = oktaClient.GetUsersClient();
 
-        //    return tenant;
-        //}
+                if (!string.IsNullOrWhiteSpace(strUserID))
+                {
+                    Models.User existingUser = usersClient.Get(oktaTenant.TestUserId);
+
+                    Assert.IsFalse(existingUser == null, "The user with id {0} doesn't exist", strUserID);
+
+                    Models.LoginCredentials loginCreds = new Models.LoginCredentials();
+                    loginCreds.Password.Value = Helpers.GetRandomString();

```

Comment:
Done this way specifically because I had no way of resetting the environment to a known state (for instance, without that user in the first place). Now possible with the user delete feature, but previously, this is how I worked around it in this context.